src/nunjucks/moduk/components/details/__examples__/open.njk

```diff
@@ -0,0 +1,7 @@
+{% from "moduk/components/details/macro.njk" import modukDetails %}
+
+{{ modukDetails({
+  summaryText: "Help with organisation",
+  text: "We need to know the organisation you work for so we can forward your request to the correct department.",
+  open: true
+}) }}
```

Comment:
What do you think of the copy? I attempted to change it to be a bit more relevant.

Reply:
I think changing 'department' to 'team' would be good enough for now.